usecase/implementation/src/test/java/eu/openg/guessnumberapi/usecase/implementation/GuessNumberInteractorTest.java

```diff
@@ -0,0 +1,83 @@
+package eu.openg.guessnumberapi.usecase.implementation;
+
+import eu.openg.guessnumberapi.domain.GameEntity;
+import eu.openg.guessnumberapi.gateway.api.GameEntityRepository;
+import eu.openg.guessnumberapi.usecase.api.BoundaryGuessResponse;
+import eu.openg.guessnumberapi.usecase.api.GuessNumberUseCase;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class GuessNumberInteractorTest {
+    @Mock
+    private GuessValidator gateway;
+    @Mock
+    private GameEntityRepository gameEntityRepository;
+
+    private GuessNumberUseCase guessNumberInteractor;
+
+    @BeforeEach
+    void setUp() {
+        guessNumberInteractor = new GuessNumberInteractor(gateway, gameEntityRepository);
+    }
+
+    @Test
+    void checkLowerGuessAndReturnResponse() {
+        checkGuessAndReturnResponse(1, false, false,
+                "Lower");
```

Comment:
čia į eilutę telpa. tau per ide atrodė, kad netelpa, nes tau rodė paramų pavadinimus, bet git to nesimato ir kiti galbūt išjungę ir nerodo jiems, todėl neturėtum atsižvelgt tik į savo ide settingus